extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -79,9 +81,21 @@ function overrideSlotService(slotRegistry, legacyBtfBlocker) {
 	slotService.legacyEnabled = slotService.enable;
 	slotService.enable = (slotName) => {
 		legacyBtfBlocker.unblock(slotName);
+		slotRegistry.enable(slotName);
+	};
+	slotService.disable = (slotName) => {
+		slotRegistry.disable(slotName);
 	};
 }
 
+function syncSlotsStatus(slotRegistry, slotsInContext) {
```

Comment:
It would be great to get rid of that sync and slotRegistry. Do we have a separate ticket for that?